recipes-kernel/linux/files/patches-5.10/0048-arm64-dts-ti-iot2050-Prepare-for-adding-2nd-generati.patch

```diff
@@ -14,16 +14,16 @@ dts files.
 
 Signed-off-by: Jan Kiszka <jan.kiszka@siemens.com>
 ---
- .../dts/ti/k3-am65-iot2050-common-pg1.dtsi    | 46 +++++++++++++++
- .../boot/dts/ti/k3-am65-iot2050-common.dtsi   | 35 +-----------
- ...ts => k3-am6528-iot2050-basic-common.dtsi} | 12 +---
- .../boot/dts/ti/k3-am6528-iot2050-basic.dts   | 56 +++----------------
- ...=> k3-am6548-iot2050-advanced-common.dtsi} |  8 +--
- .../dts/ti/k3-am6548-iot2050-advanced.dts     | 50 +++--------------
- 6 files changed, 67 insertions(+), 140 deletions(-)
+ .../dts/ti/k3-am65-iot2050-common-pg1.dtsi    | 46 ++++++++++++++
+ .../boot/dts/ti/k3-am65-iot2050-common.dtsi   | 35 +----------
+ .../ti/k3-am6528-iot2050-basic-common.dtsi    | 60 +++++++++++++++++++
+ .../boot/dts/ti/k3-am6528-iot2050-basic.dts   | 56 +++--------------
+ .../ti/k3-am6548-iot2050-advanced-common.dtsi | 56 +++++++++++++++++
+ .../dts/ti/k3-am6548-iot2050-advanced.dts     | 50 +++-------------
+ 6 files changed, 178 insertions(+), 125 deletions(-)
  create mode 100644 arch/arm64/boot/dts/ti/k3-am65-iot2050-common-pg1.dtsi
- copy arch/arm64/boot/dts/ti/{k3-am6528-iot2050-basic.dts => k3-am6528-iot2050-basic-common.dtsi} (80%)
- copy arch/arm64/boot/dts/ti/{k3-am6548-iot2050-advanced.dts => k3-am6548-iot2050-advanced-common.dtsi} (85%)
+ create mode 100644 arch/arm64/boot/dts/ti/k3-am6528-iot2050-basic-common.dtsi
+ create mode 100644 arch/arm64/boot/dts/ti/k3-am6548-iot2050-advanced-common.dtsi
```

Comment:
Seems we have different git configurations causing this delta here - or did you actually change anything in this patch?